src/utils/daos_hdlr.c

```diff
@@ -2737,9 +2737,19 @@ dm_connect(bool is_posix_copy,
 	 * tools
 	 */
 	if (src_file_dfs->type != POSIX) {
+		/* Need to query source max oid for non-POSIX source
+		 * containers, and the cont type to see if the source
+		 * container is POSIX, and if it is then use dfs_cont_create
+		 * to create the destination container
+		 */
 		dpe_types[0] = ca->cont_prop_layout;
 		dpe_types[1] = ca->cont_prop_oid;
 
+	 	/* This will be extended to get all props
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs